release/downloadIstioCandidate.sh

```diff
@@ -42,35 +42,57 @@ if [ "x${ISTIO_VERSION}" = "x" ] ; then
 		  grep -v -E "(alpha|beta|rc)\.[0-9]$" | sort -t"." -k 1,1 -k 2,2 -k 3,3 -k 4,4 | tail -n 1)
 fi
 
+if [ "x${ISTIO_ARCH}" = "x" ]; then
+  ISTIO_ARCH=$(arch)
+fi
+
+#for now this supports amd64, which has arch output of x86_64. Thhe arch command will let us switch based on host
+ISTIO_ARCH="amd64"
+
```

Comment:
Why have the ability to set it if you are going to default anyway. Remove lines 45-48 as you aren't setting arch anyway, and it wouldn't matter if you did.

Reply:
Updated to support multiple architectures. Copied from https://github.com/istio/istio/blob/abe49dfacd8b0c222e2dc2424fdad8187e01baf9/common/scripts/setup_env.sh#L28 with some changes to:
* validate user architecture entries
* allow users to enter either the `uname -m` name or common name for an architecture
* make shell check happy